src/main/java/app/chatbot/Parser.java

```diff
@@ -0,0 +1,99 @@
+package app.chatbot;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import app.command.AddCommand;
```

Comment:
Good ordering and grouping of import statements. Good use of class names as well.